jekyll/_layouts/classic-docs.html

```diff
@@ -41,8 +41,20 @@
 
             <div class="quick-info-list">
               <time aria-describedby="tooltip-time" id="time-posted-on" datetime="{{  page.last_modified_at | date_to_xmlschema }}">
-                <img src="{{ site.baseurl }}/assets/img/docs/duration.svg">
-                {% if page.last_modified_at %}<span id="time-posted-ago"></span>{% else %} {{ page.date | date: '%d-%b-%y' }} {% endif %}
+                <img alt="Clock" src="{{ site.baseurl }}/assets/img/docs/duration.svg">
+                {% if page.last_modified_at %}
+                  {% assign dateStart = page.last_modified_at | date: '%s' %}
+                  {% assign nowTimestamp = 'now' | date: '%s' %}  
+                  {% assign timestampDiff = nowTimestamp | minus: dateStart %}
+                  {% assign yearInSeconds = 60 | times: 60 | times: 24 | times: 365 %}
+                  {% if timestampDiff > yearInSeconds %}
+                    1+ year ago
+                  {% else %}  
+                    {{ page.last_modified_at | timeago }} 
+                  {% endif %} 
+                {% else %} 
+                  {{ page.date | date: '%d-%b-%y' }} 
+                {% endif %}
```

Comment:
I _think_ `page.date` will be the generation date which will be counter-intuitive since the tooltip still says `last updated`. Have we considered not showing anything at all?

Reply:
Ill ask @danbeere ! :)